python/lsst/ip/isr/linearize.py

```diff
@@ -679,6 +680,52 @@ def __call__(self, image, **kwargs):
             return False, 0
 
 
+class LinearizeSpline(LinearizeBase):
+    """Correct non-linearity with a spline model.
+
+    corrImage = uncorrImage - Spline(coeffs, uncorrImage)
+
+    Notes
+    -----
+
+    The spline fit calculates a correction as a function of the
+    expected linear flux term.  Because of this, the correction needs
+    to be subtracted from the observed flux.
+
+    """
+    LinearityType = "Spline"
+
+    def __call__(self, image, **kwargs):
+        """Correct for non-linearity.
+
+        Parameters
+        ----------
+        image : `lsst.afw.image.Image`
+            Image to be corrected
+        kwargs : `dict`
+            Dictionary of parameter keywords:
+            ``"coeffs"``
+                Coefficient vector (`list` or `numpy.array`).
+            ``"log"``
+                Logger to handle messages (`lsst.log.Log`).
+
+        Returns
+        -------
+        output : `bool`
+            If true, a correction was applied successfully.
```

Comment:
You say you return a single bool, but actually return `True, 0`.

Reply:
I wonder how many reviews have missed that.  Fixed the docstrings to match reality.